core/domain/image_validation_services.py

```diff
@@ -38,7 +38,8 @@ def validate_image_and_filename(raw_image, filename):
             validation checks.
     """
     hundred_kb_in_bytes = 100 * 1024
-
+    if isinstance(raw_image, str):
+        raw_image = raw_image.encode('ISO-8859-1')
```

Comment:
Why is this needed?

Reply:
The image that is passed as an argument to this function should be of type bytes in order to validate successfully. Since I have `encoded` all images of [Suggestion](https://github.com/chiragbaid7/oppia/blob/a80d32e3854e8a7dbc8bc20a9c67768ec54c24ef/core/controllers/suggestion_test.py#L723) and [UpdateQuestionSuggestion](https://github.com/chiragbaid7/oppia/blob/a80d32e3854e8a7dbc8bc20a9c67768ec54c24ef/core/controllers/suggestion_test.py#L976) handlers in `suggestion_test.py` and implement decoding in [test_utils.py](https://github.com/oppia/oppia/pull/14063#discussion_r743936332) .So I had to add a logic in this function to check image type coming from various handlers and perform encoding if it is a string.